src/main/java/net/mcreator/element/GeneratableElement.java

```diff
@@ -42,7 +42,7 @@
 
 public abstract class GeneratableElement {
 
-	public static final int formatVersion = 56;
+	public static final int formatVersion = 58;

```

Comment:
Why isn't it 57?

Reply:
Because I assume one of other FV-breaking PRs is going to be merged before this one, and there are not too many such PRs opened in total.